poc_iot_injector/migrations/3_files_processed.sql

```diff
@@ -0,0 +1,11 @@
+create table files_processed (
+	file_name varchar primary key,
+	file_type varchar,
```

Comment:
Do we want to make `file_type`, `file_timestamp` and `processed_at` not null as well?